integration_tests/base_routes.py

```diff
@@ -820,6 +820,9 @@ def main():
     class BasicAuthHandler(AuthenticationHandler):
         def authenticate(self, request: Request) -> Optional[Identity]:
             token = self.token_getter.get_token(request)
+            if token is not None:
```

Comment:
This file is used only for tests? I wanted to call the `set_token` method somewhere in the tests to avoid having it break after some changes without us noticing. In a production scenario the lines I added here have no interest indeed

Reply:
My bad. Sorry, this was an oversight. 😅